Resources/views/Collector/db.html.twig

```diff
@@ -411,3 +434,238 @@
+            </div>
+        {% else %}
+            <table class="auto-layout sql-queries" id="queriesPlaceholder-{{ loop.index }}">
+                <thead>
+                    <tr>
+                        <th class="nowrap" onclick="javascript:sortTable(this, 0, 'queries-{{ loop.index }}')" data-sort-direction="-1" style="cursor: pointer;"># <span>&#9650;</span></th>
+                        <th class="nowrap" onclick="javascript:sortTable(this, 1, 'queries-{{ loop.index }}')" style="cursor: pointer;">Time <span></span></th>
+                        <th>Info</th>
+                    </tr>
+                </thead>
+                <tbody id="queries-{{ loop.index }}">
+                {% for i, query in queries %}
+                    <tr id="queryNo-{{ i }}-{{ loop.parent.loop.index }}">
+                        <td>{{ loop.index }}</td>
+                        <td class="nowrap">{{ '%0.2f'|format(query.executionMS * 1000) }} ms</td>
+                        <td>
+                            <div class="sql-section">
+                                {{ query.sql|doctrine_highlight_sql }}
+
+                                <span class="sql-parameters newline small-text">
+                                    <strong>Parameters</strong>: {{ query.params|yaml_encode }}
+                                </span>
+                            </div>
+
+                            <div class="sql-actions">
+                                <a href="#" id="expandParams-{{ i }}-{{ loop.parent.loop.index }}" onclick="javascript:toggleRunnableQuery(this); return false;" data-target-id="original-query-{{ i }}-{{ loop.parent.loop.index }}">View runnable query</a>
+
+                                {% if query.explainable %}
+                                    <a href="{{ path('_profiler', { panel: 'db', token: token, page: 'explain', connection: connection, query: i }) }}" onclick="return explain(this);" data-target-id="explain-{{ i }}-{{ loop.parent.loop.index }}">Explain query</a>
+                                {% endif %}
+                            </div>
+
+                            <span id="original-query-{{ i }}-{{ loop.parent.loop.index }}" class="sql-runnable">
+                                {{ (query.sql ~ ';')|doctrine_replace_query_parameters(query.params, highlight = false)|doctrine_highlight_sql }}
+                            </span>
+
+                            {% if query.explainable %}
+                                <div id="explain-{{ i }}-{{ loop.parent.loop.index }}"></div>
+                            {% endif %}
+                        </td>
+                    </tr>
+                {% endfor %}
+                </tbody>
+            </table>
+        {% endif %}
+    {% endfor %}
+
+    <h2>Database Connections</h2>
+
+    {% if collector.connections %}
+        {{ include('WebProfilerBundle:Profiler:table.html.twig', { data: collector.connections, labels: ['Name', 'Service'] }, with_context = false ) }}
```

Comment:
I suggest switching to ``@WebProfiler/...`` template names for consistency with WebProfilerBundle recommendations (even though DoctrineBundle is not usable outside Symfony to display a Doctrine profiler currently)